pterm.go

```diff
@@ -6,20 +6,22 @@
 // View the animated examples here: https://github.com/pterm/pterm#-examples
 package pterm
 
-import "github.com/gookit/color"
+import (
+	"github.com/gookit/color"
+	"go.uber.org/atomic"
+)
 
 var (
 	// Output completely disables output from pterm if set to false. Can be used in CLI application quiet mode.
-	Output = true
+	Output = atomic.NewBool(true)
 
 	// PrintDebugMessages sets if messages printed by the DebugPrinter should be printed.
-	PrintDebugMessages = false
+	PrintDebugMessages = atomic.NewBool(false)
 
 	// RawOutput is set to true if pterm.DisableStyling() was called.
 	// The variable indicates that PTerm will not add additional styling to text.
 	// Use pterm.DisableStyling() or pterm.EnableStyling() to change this variable.
-	// Changing this variable directly, will disable or enable the output of colored text.
-	RawOutput = false
+	RawOutput = atomic.NewBool(false)
```

Comment:
I totally agree with your explanation. Unfortunately, this is used in many projects to map CLI flags directly to the variable via a pointer. We can do this, but we would definitely need to release it under a breaking change and increase to `v0.13.0`.

![image](https://user-images.githubusercontent.com/31022056/214692150-c5c08f22-1017-4caf-8f87-80ee923ab365.png)
![image](https://user-images.githubusercontent.com/31022056/214692221-8ecc540f-2788-44eb-bb2c-ae515ff33615.png)

Reply:
Ah shoot great point, makes sense as a use-case too. Let me think on this more. Might revisit later tonight. I think I can implement this to work & use a Mutex lock for each one so the helper methods are thread-safe. Only use it like that in the tests & such. Then just have it documented (like it is now) that direct usage is not recommended as its not 100% thread-safe.

Reply:
Just FYI I would totally be fine with releasing `v0.13.0` We had 54 releases without a breaking change. I would just throw in some other breaking changes that pilled up before the release, and we'd be good to go. The exposed variables also come from a time when PTerm was not intended to be used by others.

Reply:
Sounds good I'll let you know what I come up with here. I think this can be better regardless so I don't want to jump to a breaking change if it doesn't add value.